sample/hash_to_array.rb

```diff
@@ -0,0 +1,3 @@
+def hashToArray(hash)
```

Comment:
Why is the method name [camel case](https://en.wikipedia.org/wiki/Camel_case) rather than [snake case](https://en.wikipedia.org/wiki/Snake_case) here?